operations/firmware/FAS_Use_Cases.md

```diff
@@ -760,74 +673,262 @@ Use this procedure to update compute node BIOS firmware using FAS. There are two
+or rerunning FAS to verify that the BMC firmware was updated.
+Make sure you have waited for the current firmware to be updated before starting a new FAS action on the same node.
+
+**Device Type: NodeBMC | Target: BIOS**
+```json
+{
+"stateComponentFilter": {
+
+    "deviceTypes": [
+      "nodeBMC"
+    ],
+    "xnames": [
+      "x3000c0s1b0"
+    ]
+},
+"inventoryHardwareFilter": {
+    "manufacturer": "gigabyte"
+    },
+"targetFilter": {
+    "targets": [
+      "BIOS"
+    ]
+  },
+"command": {
+    "version": "latest",
+    "tag": "default",
+    "overrideDryrun": false,
+    "restoreNotPossibleOverride": true,
+    "timeLimit": 4000,
+    "description": "Dryrun upgrade of Gigabyte node BIOS"
+  }
+}
+```
+
+**IMPORTANT**: The *timeLimit* is `4000` because the Gigabytes can take a lot longer to update.
+
+#### HPE
+
+**Device Type: NodeBMC | Target: `iLO 5` aka BMC**
+
+```json
+{
+"stateComponentFilter": {
+    "deviceTypes": [
+      "nodeBMC"
+    ],
+    "xnames": [
+      "x3000c0s1b0"
+    ]
+},
+"inventoryHardwareFilter": {
+    "manufacturer": "hpe"
+    },
+"targetFilter": {
+    "targets": [
+      "iLO 5"
+    ]
+  },
+"command": {
+    "version": "latest",
+    "tag": "default",
+    "overrideDryrun": false,
+    "restoreNotPossibleOverride": true,
+    "timeLimit": 1000,
+    "description": "Dryrun upgrade of HPE node iLO 5"
+  }
+}
+```
+
+#### Procedure for updating NCNs:
+1. For `HPE` NCNs, check the DNS servers by running the script `/opt/cray/csm/scripts/node_management/set-bmc-ntp-dns.sh ilo -H XNAME -s`.  Replace `XNAME` with the xname of the NCN BMC.
+   See [Configure DNS and NTP on Each BMC](../../install/redeploy_pit_node.md#configure-dns-and-ntp-on-each-bmc") for more information.
+2. Run a dryrun for all NCNs first to determine which NCNs and targets need updating.
+3. For each NCN requiring updates to target `BMC` or `iLO 5`:
+   **NOTE:** Update of `BMC` and `iLO 5` will not affect the nodes.
+   1. Unlock the NCN BMC.
+See [Lock and Unlock Management Nodes](../hardware_state_manager/Lock_and_Unlock_Management_Nodes.md).
+   2. Run the FAS action on the NCN.
+   3. Relock the NCN BMC.
+See [Lock and Unlock Management Nodes](../hardware_state_manager/Lock_and_Unlock_Management_Nodes.md).
+4. For each NCN requiring updates to target `BIOS` or `System ROM`:
+   1. Unlock the NCN BMC.
+See [Lock and Unlock Management Nodes](../hardware_state_manager/Lock_and_Unlock_Management_Nodes.md).
+   2. Run the FAS action on the NCN.
+   3. Reboot the Node.
+   See [Reboot NCNs](../node_management/Reboot_NCNs.md).
+   4. For `HPE` NCNs, run the script `/opt/cray/csm/scripts/node_management/set-bmc-ntp-dns.sh`.
+   See [Configure DNS and NTP on Each BMC](../../install/redeploy_pit_node.md#configure-dns-and-ntp-on-each-bmc").
+   5. Relock the NCN BMC.
+See [Lock and Unlock Management Nodes](../hardware_state_manager/Lock_and_Unlock_Management_Nodes.md).
+
+**Device Type: NodeBMC | Target: `System ROM` aka BIOS**
```

Comment:
Should this example file also be moved above the procedure section?

Reply:
Moved